hw_3/Task3.js

```diff
@@ -0,0 +1,8 @@
+/*
+Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+n + nn + nnn, где n не перемножаются, а конкатенируются
+*/
+
+let n = 1;
```

Comment:
Can add checks for `n` variable value using `if...else` or the ternary operator (if u saw second lection)

Reply:
Done